### ISSUE TITLE
Cmder hangs after getting initialized on Windows 10

### ISSUE THREAD

**Issue by itsraghz** [cmderdev/cmder#2320]
### Purpose of the issue
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question

### Version Information

Version 1.3.14.982
Windows 10 Home Edition

### Description of the issue

Cmder with the new edition (Version 1.3.14.982) on Windows 10 just hangs after it is initialized. not really sure what happens. After reading some issue threads in github for this account, I tried unchecking the Read Enhancement option on both keyboard and mouse but no luck. FYI, I had been trying out with keyboard only so far.

**Comment by itsraghz** [comment]
Hey, the issue is resolved in my case. When I was exploring the settings during the initial booting time of Cmder, there is a feature called "Enable Automatic Updates" having 3 different radio buttons - (1) Stable (2) Preview (3) 'Alpha' builds, with a default selection on 'Preview'. I changed this to 'Stable' and clicked on. I got the prompt to type and it works fine now :) 

![image](https://user-images.githubusercontent.com/1222999/83072379-4c0ce300-a08c-11ea-9c7b-ac2ef0957a68.png)

![image](https://user-images.githubusercontent.com/1222999/83072482-79599100-a08c-11ea-9984-ac2d68b34cb7.png)

Not sure it is the right/appropriate one. However, this option appears after you restart the PC and initialize Cmder again.
